config/environments/deployed_development.rb

```diff
@@ -40,14 +40,15 @@
   config.active_storage.service = :local
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
Clearly this _does_ work, but I'm surprised. The deployed app communicates with a CDN/router/load balancer/something over port 8080 using HTTP, not HTTPS. 🤷 so long as it works